airbyte-ci/connectors/pipelines/pipelines/bases.py

```diff
@@ -18,20 +18,30 @@
 import anyio
 import asyncer
 from anyio import Path
-from connector_ops.utils import Connector, console
+from connector_ops.utils import CDK, Connector, console
 from dagger import Container, DaggerError
 from jinja2 import Environment, PackageLoader, select_autoescape
-from pipelines import sentry_utils
-from pipelines.actions import remote_storage
-from pipelines.consts import GCS_PUBLIC_DOMAIN, LOCAL_REPORTS_PATH_ROOT, PYPROJECT_TOML_FILE_PATH
-from pipelines.utils import METADATA_FILE_NAME, check_path_in_workdir, format_duration, get_exec_result
 from rich.console import Group
 from rich.panel import Panel
 from rich.style import Style
 from rich.table import Table
 from rich.text import Text
 from tabulate import tabulate
 
+from pipelines import sentry_utils
+from pipelines.actions import remote_storage
+from pipelines.consts import (
```

Comment:
i haven't run this through black/flake yet so this is just my local machine doing things that don't match your formatting settings